### PR TITLE
DOCSP-45198: Distinct values

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-45198

### Staging Links

<li><a href=https://deploy-preview-111--docs-ruby.netlify.app/read/distinct>read/distinct</a></li><li><a href=https://deploy-preview-111--docs-ruby.netlify.app/read>read</a></li>

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [ ] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/read/distinct.txt [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+.. meta::
+   :keywords: read, unique, code example
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+driver-short+} to retrieve the
+distinct values of a specified field across a collection.
+
+Within a collection, different documents might contain different values for a
```
Comment (rustagir) [inline source/read/distinct.txt:26]:
```suggestion
Within a collection, documents might contain different values for a
```

File: source/read/distinct.txt [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+single field. For example, one document in a ``restaurants`` collection has a
+``borough`` value of ``'Manhattan'``, and another has a ``borough`` value of
+``'Queens'``. By using the {+driver-short+}, you can retrieve all the unique values
+that a field contains across multiple documents in a collection.
```
Comment (rustagir) [inline source/read/distinct.txt:30]:
```suggestion
``'Queens'``. You can use the {+driver-short+} to retrieve all the unique values
that a field contains across multiple documents in a collection.
```

File: source/read/distinct.txt [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+~~~~~~~~~~~
+
+The examples in this guide use the ``restaurants`` collection in the ``sample_restaurants``
+database from the :atlas:`Atlas sample datasets </sample-data>`. To access this collection
+from your {+language+} application, create a ``Mongo::Client`` object that connects to
+an Atlas cluster and assign the following values to your ``database`` and ``collection``
+variables:
+
+.. literalinclude:: /includes/read/distinct.rb
+    :language: ruby
```
Comment (rustagir) [inline source/read/distinct.txt:42]: S: fix indentation

File: source/read.txt [hunk 4]
```diff
@@ -25,3 +25,4 @@ Read Data from MongoDB
    Retrieve Data </read/retrieve>
    Specify Documents to Return </read/specify-documents-to-return>
    Specify Fields to Return </read/project>
+   Distinct Field Values </read/project>
```
Comment (rustagir) [inline source/read.txt:28]:
```suggestion
   Distinct Field Values </read/distinct>
```

**Review by rustagir — APPROVED** [review]
few small things but dont need to take another look

**Review by comandeo-mongo — APPROVED** [review]

**Pull request merged**